content/telegraf/v1/release-notes-changelog.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Telegraf 1.27 release notes
+title: Telegraf 1.28 release notes
```

Comment:
Should the version be included here?

Reply:
I think we should just remove the version